### PR TITLE
link yaml-cpp statically

### PR DESCRIPTION
**Pull request by christian-rauch**
link yaml-cpp statically so we do not have to install the yaml-cpp library manually

### CODE REVIEW

File: src/project_manager/CMakeLists.txt [hunk 1]
```diff
@@ -3,11 +3,19 @@ set(CMAKE_AUTORCC ON)
...
 find_package(QtCreator COMPONENTS Core REQUIRED)
 find_package(QT NAMES Qt6 Qt5 COMPONENTS Widgets REQUIRED)
 find_package(yaml-cpp REQUIRED)
 find_package(qtermwidget5 REQUIRED)
 
+message("YAMLCPP_FOUND:" ${YAMLCPP_FOUND})
+message("YAMLCPP_LIBRARY:" ${YAMLCPP_LIBRARY})
+message("YAMLCPP_LIBRARY_DIR:" ${YAMLCPP_LIBRARY_DIR})
+message("YAMLCPP_USE_STATIC_LIBS:" ${YAMLCPP_USE_STATIC_LIBS})
+message("YAMLCPP_STATIC:" ${YAMLCPP_STATIC})
```
Comment (Levi-Armstrong) [inline src/project_manager/CMakeLists.txt]: Do you want to leave these here or should they be removed?

Comment (christian-rauch) [reply src/project_manager/CMakeLists.txt]: They should be removed. I added them temporarily to debug a linking issue on macOS.

**Comment by christian-rauch** [comment]
This requires https://github.com/jbeder/yaml-cpp/pull/1032 for building yaml-cpp with position independent code. I will keep this open as draft until the yaml-cpp fix has been merged upstream.

**Comment by christian-rauch** [comment]
Obsolete with https://github.com/ros-industrial/ros_qtc_plugin/pull/511.

**Pull request closed without merging**